site/docs/providers/cometapi.md

```
@@ -0,0 +1,135 @@
+node dist/src/main.js eval -c examples/cometapi/promptfooconfig.yaml
+```
+
+## Setup
+
+Set your API key as an environment variable:
+
+```bash
+export COMETAPI_KEY=<your_api_key>
+```
+
+If you don’t have an API key yet, get one at 👉 https://api.cometapi.com/console/token
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)